server/utils/auth/rbac_decorators.py

```
@@ -25,6 +25,16 @@
 logger = logging.getLogger(__name__)
 
 
+def _audit_auth_failure(user_id, org_id, action, detail):
+    """Best-effort audit log for auth/RBAC failures."""
+    try:
+        from routes.audit_routes import record_audit_event
+        from flask import request as _req
+        record_audit_event(org_id or "", user_id or "", action, "auth", None, detail, _req)
+    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)